Kubernetes/helmcharts/azure-policy-addon-arc-connected-clusters/templates/gatekeeper.yaml

```diff
@@ -320,7 +320,7 @@ spec:
           valueFrom:
             fieldRef:
               fieldPath: metadata.name
-        image: mcr.microsoft.com/oss/open-policy-agent/gatekeeper:v3.1.0-beta.6
+        image: mcr.microsoft.com/oss/open-policy-agent/gatekeeper:v3.1.0-beta.7
```

Comment:
whenever we upgrade the image tag, please also make sure the yaml has all the changes as well. I see it's currently missing few things.